docs/markdown/android/overlaypanel.md

```diff
@@ -0,0 +1,115 @@
+---
+title: OverlayPanel
+description: OverlayPanels are bottom-anchored overlays that that allow the user to easily return to the previous screen. They are meant for temporary focused tasks. OverlayPanels are the mobile equivalent of the modal or overlay on web.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/41/a6/36/41a6369d262d24bf04d08ee32beef508.png" alt="examples of three types of overlay panels: a Full overlay panel, a Partial resizable overlay panel and an Action overlay panel"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />
+      - Performing an optional sub-task within a larger task
+      - Presenting help info while maintaining the current page and its context
+      - Requesting minimal amounts of information froma user (1-2 fields only)
+      - Capturing user’s full attention for something important
+  </Group>
+  <Group>
+    <Dont title="When not to use" />
+      - Displaying system errors or notices. Consider a [Callout](callout) instead
+      - Any time a separate, designated URL is desired
+      - Any action that should not interrupt users from their current work stream
+      - On top of another overlay panel, as this can create usability issues and confusion
+  </Group>
+</TwoCol>
+
+## Mobile best practices
```

Comment:
We're inconsistent with this across our mobile components, but "Mobile" is redundant here, yes? We're already in the context of a mobile component, so it doesn't seem like it needs to be specified

Reply:
UNDOING